src/toady/resolve_service.py

```
@@ -52,12 +51,47 @@
+                        "unauthorized",
+                    ]
+                    for msg in error_messages
+                ):
+                    raise ThreadPermissionError(
+                        f"Permission denied - cannot resolve thread {thread_id}: "
+                        f"{'; '.join(error_messages)}"
+                    )
+                else:
+                    raise ResolveServiceError(
+                        f"Failed to resolve thread: {'; '.join(error_messages)}"
+                    )
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "raise", remove the leading "el" from "elif"

(R1720)